nano-defaults.el

```diff
@@ -62,10 +62,10 @@
 ;; Moderate font lock
 (setq font-lock-maximum-decoration nil)
 
-;; No limite on font lock
+;; No limit on font lock
 (setq font-lock-maximum-size nil)
 
-;; No line breat space points
+;; No line break space points
 (setq auto-fill-mode nil)
```

Comment:
@rougier I guess this should be `(auto-fill-mode -1)` or something? There's no _variable_ ending in `-mode` with this name.

Reply:
Yes, thank you for having spotted it. Can you add the change in your PR ?